app/assets/javascripts/discourse/app/components/d-editor.js

```diff
@@ -231,6 +238,8 @@ export default Component.extend(TextareaTextManipulation, {
   ready: false,
   lastSel: null,
   _itsatrap: null,
+  _previewElement: null,
+  _morphFunction: null,
```

Comment:
Caches the morphing function so it's only loaded once instead of on every (debounced) keystrokes.